packages/themes/bmf/src/components/input-radio.scss

```diff
@@ -156,9 +156,9 @@
 			appearance: none;
 			transition: 0.5s;
 			border-radius: 100%;
-			height: calc(6 * var(--spacing));
-			min-width: calc(6 * var(--spacing));
-			width: calc(6 * var(--spacing));
+			height: 1.5; // calc(6 * var(--spacing));
+			min-width: 1.5; // calc(6 * var(--spacing));
+			width: 1.5; // calc(6 * var(--spacing));
```

Comment:
Ungültige Werte, haben gerade keinen Effekt.

Könnten wir bei der Gelegenheit auch noch den auskommentierten Code aus der Datei entfernen?

Reply:
ist gefixt, auch dies ist im Grunde in allen Themes das gleiche